app/controllers/idv/review_controller.rb

```diff
@@ -124,6 +125,13 @@ def password
       params.fetch(:user, {})[:password].presence
     end
 
+    def confirm_verify_info_complete
```

Comment:
Is this related to the changes for string redaction?

Reply:
Nope, base branch issues like @zachmargolis pointed out below.